lib/jxl/modular/encoding/enc_debug_tree.cc

```diff
@@ -115,10 +111,6 @@ void PrintTree(const Tree &tree, const std::string &path) {
   }
   fprintf(f, "}\n");
   fclose(f);
-#if JXL_ENABLE_DOT
-  JXL_ASSERT(
```

Comment:
I would not remove this code, it is very useful for debugging. A JXL_CHECK is fine here.

Reply:
IMHO that is not nice to run other binaries from our code; and likely not portable;
Perhaps move that functionality to ci.sh?

Reply:
JXL_ENABLE_DOT is never enabled - it's just a debug option, it should only be enabled by JXL devs debugging stuff ;)

Reply:
rolled back